qa/suites/rgw/lua/tasks/0-install.yaml

```diff
@@ -0,0 +1,22 @@
+tasks:
+- install:
+- ceph:
+- openssl_keys:
+- rgw:
+    client.0:
+
+overrides:
+  install:
+    ceph:
+      extra_system_packages:
+        rpm:
+        - luarocks
+        deb:
+        - luarocks
```

Comment:
if this is required to use lua with radosgw, shouldn't that be a package dependency?

i see it as a *build* dependency in ceph.spec.in and debian/control, but it should be a runtime one